src/main/java/org/schemaspy/LayoutFiles.java

```diff
@@ -0,0 +1,48 @@
+package org.schemaspy;
+
+import org.apache.commons.io.filefilter.FileFilterUtils;
+import org.apache.commons.io.filefilter.IOFileFilter;
+import org.schemaspy.util.ResourceWriter;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.Objects;
+
+/**
+ * Represents the contents of the layout folder.
+ */
+public class LayoutFiles {
+
+    private final ClassLoader classLoader;
+    private final String DOT_HTML;
```

Comment:
Either take a FileFilter or rename to something better.

`DOT_HTML` was a constant now it isn't, it's used in a suffix Filter, and it was an extension.